### PR TITLE
5176 name translation summary page

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#5176

*Description of changes:*

- additional ux changes for this ticket
- upper case should be for display only, and stored as the user entered it

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
Please fix unit tests.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "0.5.27",
+  "version": "0.5.29",
```
Comment (severinbeauvais) [inline package.json:3]: This is correct but you'll get a merge conflict because GH doesn't know whether John's PR is correct or yours.

**Comment by jordiwes** [comment]
I have updated the tests and slipped in a couple of ux changes to document delivery as well

File: public/config/configuration.json [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
   "AUTH_URL": "https://dev.bcregistry.ca/business/auth/",
   "DASHBOARD_URL": "https://dev.bcregistry.ca/business/",
-  "LEGAL_API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/",
-  "AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1/",
-  "SBC_CONFIG_AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1",
-  "PAY_API_URL": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/",
+  "LEGAL_API_URL": "https://legal-api-dev.apps.silver.devops.gov.bc.ca/api/v1/",
+  "AUTH_API_URL": "https://auth-api-dev.apps.silver.devops.gov.bc.ca/api/v1/",
+  "SBC_CONFIG_AUTH_API_URL": "https://auth-api-dev.apps.silver.devops.gov.bc.ca/api/v1",
+  "PAY_API_URL": "https://pay-api-dev.apps.silver.devops.gov.bc.ca/api/v1/",
```
Comment (severinbeauvais) [inline public/config/configuration.json:8]: Looks fine but I already updated these, causing you a conflict (sorry). I did text you in RC to see what project(s) you were updating but didn't see a response so I went ahead and updated all projects.

Comment (jordiwes) [reply public/config/configuration.json]: I resolved the conflict

**Review by severinbeauvais — APPROVED** [review]
Looks good.
The new tests look great!

**Comment by severinbeauvais** [comment]
Good to merge after checks pass!

**Pull request merged**